frontend/projects/valtimo/components/assets/css/carbon.scss

```
@@ -122,7 +118,7 @@ $darkThemeOverwriteVariables: (
   }
 }
 
-@import 'node_modules/@carbon/charts-angular/styles.min.css';
+@import '~node_modules/@carbon/charts-angular/styles.min.css';
```

Stylelint (17.11.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)